### ISSUE TITLE
Can't do unauthenticated UDP connections

### ISSUE THREAD

**Issue by maxha651** [ValveSoftware/GameNetworkingSockets#244]
I'm trying to get unauthenticated UDP connections to works and afaict it should be possible to have a direct connection without logging in to steam or any internet access as long as AllowWithoutAuth is set. Correct?

I don't get any immediate error, but the server (which called CreateListenSocketIP) is continuously getting the error "We don't know our local identity." when trying to accept the incoming connection. I looked a bit at the code and I suspect that while the connecting client sets the local identity [here](https://github.com/ValveSoftware/GameNetworkingSockets/blob/14e894d6736661d28bb786bc74e05c3d9a8a4449/src/steamnetworkingsockets/clientlib/steamnetworkingsockets_udp.cpp#L1154), that part seems missing for the server?

I'm using the Steamworks SDK, not the open source version if it makes any difference.

**Comment by maxha651** [comment]
Yeah, sorry I think I was a bit unclear. What I am looking for is for the server to support non-SDR unauthenticated connections as well as authenticated SDR connections.

The use-case is to have the server connect via SDR by default but still allow unauthenticated LAN connections without internet.

**Comment by zpostfacto** [comment]
Apologies for the very late response on this. For anyone finding this later: I believe `k_ESteamNetworkingConfig_IP_AllowWithoutAuth` should address this use case. If it doesn't work for you, please let me know.

There have been some bugfixes with unauthenticated connections.  It's been so long.....sorry about that.

**Comment by maxha651** [comment]
Hi, I almost forgot about this, I did find some time now to test this. Seems to work well for us 👍 

Big thanks for following up on this!
